.github/workflows/pytest.yaml

```diff
@@ -0,0 +1,23 @@
+name: pytest
+on:
+  pull_request:
+    types: [opened, reopened, synchronize]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+jobs:
+  run-tests:
+    runs-on: ARM64
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-python@v5
+        with:
+          python-version: '3.11'
```

Comment:
Can this be pointed at the Pipfile so the version doesn't need updating in 2 places whenever it's time to upgrade past 3.11?

Reply:
I can't think of an easy way to do this. I would also make the argument that if we bump to 3.12 we would want to make sure we dont have regressions in 3.11 so it's possible this LoC could stay